### PR TITLE
minor docs updates

### PR DESCRIPTION
**Pull request by andrewkmin**
- Fix broken markdown link (https://docs.eigenlayer.xyz/eigenda/operator-guides/networks/goerli)
- Add additional Holesky faucets (note: I've tested these myself and they're functional, but happy to remove)

### CODE REVIEW

**Review by wesfloyd — APPROVED** [review]
lgtm! Thanks for the suggestions

File: docs/eigenda/operator-guides/networks/goerli.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ sidebar_position: 1
 # Goerli (Deprecated)
 
 The Goerli network has been deprecated. If you are interested in launching an
-EigenDA test operator please follow the [Holesky](./holesky.md) testnet guide.
+EigenDA test operator please follow the [Holesky](./holesky.mdx) testnet guide.
```
Comment (wesfloyd) [inline docs/eigenda/operator-guides/networks/goerli.md:7]: Can you please remove this file modification? Asking b/c it clashes with this change: https://github.com/Layr-Labs/eigenlayer-docs/pull/158/files

**Review by wesfloyd — CHANGES_REQUESTED** [review]
Can you please remove the modification to goerli.md? Asking b/c it clashes with this change: https://github.com/Layr-Labs/eigenlayer-docs/pull/158/files

**Comment by dabit3** [comment]
Thanks @andrewkmin if you can rebase / fix merge conflicts, we'll merge, I believe the other issues are fixed (broken link, sidebar position), so consider removing and just leaving the faucet links!

**Comment by andrewkmin** [comment]
> Thanks @andrewkmin if you can rebase / fix merge conflicts, we'll merge, I believe the other issues are fixed (broken link, sidebar position), so consider removing and just leaving the faucet links!

updated! thanks 🙏

**Review by dabit3 — APPROVED** [review]

**Pull request merged**